StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostViewController.swift

```diff
@@ -99,13 +102,29 @@ extension HostViewController {
     private func getManifest() {
         loadingView.errorView.isHidden = true
         loadingView.showLoading(true)
+
+        analyticsClient.log(
+            analytic: FinancialConnectionsSheetInitialSynchronizeStarted(clientSecret: clientSecret),
+            apiClient: .shared
+        )
+
         apiClient
             .synchronize(
                 clientSecret: clientSecret,
                 returnURL: returnURL
             )
             .observe { [weak self] result in
                 guard let self = self else { return }
+
+                analyticsClient.log(
+                    analytic: FinancialConnectionsSheetInitialSynchronizeCompleted(
+                        clientSecret: clientSecret,
+                        success: result.success,
+                        possibleError: result.error
+                    ),
+                    apiClient: .shared
```

Comment:
I believe it would be cleaner (and mostly going off precedent) to avoid the singleton:

https://github.com/stripe/stripe-ios/blob/6968678b/StripeFinancialConnections/StripeFinancialConnections/Source/FinancialConnectionsSheet.swift#L216-L221